docs/tutorial/part-zero/index.md

```diff
@@ -1,23 +1,144 @@
+- `--global`:
+    - When we install npm packages, we can install them globally, or in a specific project. (We’ll learn about the latter, later). The `--global` flag signals that we want the first option, to install globally. This means our package will be available to us on our computer, outside of the context of a specific project.
+- `gatsby-cli`:
+    - This is the exact name our package is registered with on the [npm registry](https://www.npmjs.com/package/gatsby-cli).
+
+### ✋ Check your Gatsby CLI installation
+1. Open up your terminal. 
+2. Run `gatsby --version`.
+3. Run `gatsby --help`.
+
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_8FB0E402DCBE189CA26AEC8EB1B27767B46B544EF50AE1038B4299BF5C07AD38_1520482614025_gatsby-cli.gif)
+
+
+As before (when we checked our Node.js install), the `gatsby --version` should return a version number. `--help` shows different commands available to you, now that the gatsby-cli is installed.
+
+## Generate a new site with “Hello World”
+
+Now let’s use the gatsby-cli tool to create our first Gatsby site. The Gatsby CLI tool lets you install “starters”, which are partially built sites with some default configuration to help you get moving faster on creating a certain type of site. Let’s scaffold out our first site!
+
+### ✋ Generate a new Gatsby site
+1. Open up your terminal. 
+2. Run `gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`. (*Note: Depending on your download speed, the amount of time this takes could vary*).
+3. Run `cd hello-world`.
+4. Run `gatsby develop`. *(Note: Leave this running! We'll talk more about what that means in a minute.)*
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_8FB0E402DCBE189CA26AEC8EB1B27767B46B544EF50AE1038B4299BF5C07AD38_1520718475736_part-zero-starter.gif)
+
+
+So what did we just do?
+
+- `gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`:
+    - Starting with `gatsby` says, ‘hey, we want to use the gatsby-cli tool!’
+    - `new` is a command available via the gatsby-cli to create a new Gatsby project. (remember to check out `gatsby --help`!)
+    - Here, `hello-world` is an arbitrary title — you could pick anything. The CLI tool will place the code for your new site in a folder called “hello-world”.
+    - Lastly, the Github URL we specified points to a code repository that holds the starter code we want to use.
+- `cd hello-world`: 
+    - This says ‘I want to change directories (`cd`) to this other subfolder called “hello-world”. Whenever you want to run any commands for your site, you need to be in the context for that site (aka, your terminal needs to be pointed at the directory where your site code lives).
+- `gatsby develop`:
+    - This command starts a development server. You will be able to see and interact with your new site in a development environment — local (on your computer, not published to the internet).
+
```

Comment:
This is really helpful. Command language is very different than everyday language.